yeastdnnexplorer/ml_models/iterative_culling.py

```diff
@@ -0,0 +1,545 @@
+import logging
+from typing import Any
+
+import numpy as np
+import pandas as pd
+import patsy
+import statsmodels.api as sm
+from patsy import dmatrices
+from sklearn.linear_model import LassoCV
+from sklearn.metrics import r2_score
+from sklearn.model_selection import StratifiedKFold
+
+logger = logging.getLogger("model_selection")
```

Comment:
I have two loggers configured in `__main__.py`, 'shiny' and 'main'. Outside of the shiny module, use `main`.

```suggestion
logger = logging.getLogger("main")
```